client/src/js/services/grid/GridGrouping.js

```diff
@@ -114,6 +116,13 @@ function GridGroupingService(uiGridGroupingConstants, $filter, Session, $timeout
     }
   }
 
+  function handleBatchSelection (){
```

Comment:
I want just to store the number of selected rows when the user choose every line directly by pressing the very top checkbox of the grid.